base/docs/basedocs.jl

```diff
@@ -142,6 +142,134 @@ keywords[:return] = doc"""
   block) and gives a value back to `map`. `test2([5,6,7])` then returns `[5,12,7]`.
   """
 
+keywords[:if] = doc"""
```

Comment:
Since the `if`, `elseif`, and `else` docs are quite similar could they be rolled into a single doc?

```julia
keywords[:if] = keywords[:elseif] = keywords[:else] = """...
```